packages/react-core/src/components/Wizard/examples/Wizard.md

```diff
@@ -55,322 +92,83 @@ If you seek a wizard solution that allows for more composition, see the [React n
 
 ### Validate on button press
 
-This example demonstrates how to use the `WizardContextConsumer` to consume the `WizardContext`. `WizardContext` can be used to imperatively move to a specific wizard step.
+```ts file="./WizardValidateOnButtonPress.tsx"
+```
 
-The definition of the `WizardContext` is as follows:
+### Progressive steps
 
+```ts file="./WizardProgressiveSteps.tsx"
 ```
-interface WizardContext {
-  goToStepById: (stepId: number | string) => void;
-  goToStepByName: (stepName: string) => void;
-  onNext: () => void;
-  onBack: () => void;
-  onClose: () => void;
-  activeStep: WizardStep;
-}
+
+### Get current step
+
+```ts file="./WizardGetCurrentStep.tsx"
 ```
 
-```ts file="./WizardValidateOnButtonPress.tsx"
+### Within modal
```

Comment:
@mcarrano should we move some of examples into the demo tab to match what we had before? This example was demo in old version.

Reply:
Yes, I think it makes sense to be consistent with our approach with the documentation unless there is a good reason to change it. I don't think we should have a "Deprecated demos" tab, but just update the existing demos to use the new component. Do you agree @tlabaj ?

Reply:
@mcarrano yes, I think that makes sense.

Reply:
I created #8867 as a followup to address the demos, would love to be able to get this in sooner if possible to prevent further merge conflict issues.